cmd/minishift/cmd/util/util.go

```diff
@@ -112,3 +115,44 @@ func IsValidProfileName(profileName string) bool {
 	rr := regexp.MustCompile(`^[a-zA-Z0-9]+[\w+-]*$`)
 	return rr.MatchString(profileName)
 }
+
+// RunAndGetExitCode returns the exit code of a command
+func RunAndGetExitCode(command string, args ...string) int {
```

Comment:
I don't think this is a _cmd_ specific util method. It should probably go somewhere under _pkg/util_, since it is useful in other cases as well.

Reply:
That said, there is already the Runner interface under _pkg/util/runner.go_. It looks like it already does what we need. No need to re-invent the wheel.

Reply:
Yes, earlier i thought it had extra stuff cause we use it for `OcRunner` but it does the same thing, i will modify this to make use of `Runner` here.

Reply:
+1